docs/source/triggers/index.rst

```diff
@@ -125,159 +111,316 @@ Like ``Scenario`` or ``Update``, you can use ``do`` notation and
 We can specify the templates that our trigger will operate
 on. In our case, we will simply specify ``AllInDar`` which means that
 the trigger will receive events for all template types defined in the
-DAR. It is also possible to specify an explicit list of templates,
-e.g., ``RegisteredTemplates [registeredTemplate @Original, registeredTemplate @Subscriber, registeredTemplate @Copy]``.
+DAR. It is also possible to specify an explicit list of templates. For example,
```

Comment:
Not for this PR but we might want to consider splitting this into two sections:
1. The next step section
2. all the gory details on registered templates, hearbeats, … which don’t matter for this.